pkg/plugins/golang/v3/scaffolds/webhook.go

```diff
@@ -75,19 +71,22 @@ func (s *webhookScaffolder) newUniverse() *model.Universe {
 }
 
 func (s *webhookScaffolder) scaffold() error {
-	if s.conversion {
+
+	if s.webhook.Conversion {
 		fmt.Println(`Webhook server has been set up for you.
 You need to implement the conversion.Hub and conversion.Convertible interfaces for your CRD types.`)
 	}
 
-	s.config.UpdateResources(s.resource.GVK())
+	res := s.resource.Data()
+	res.Webhooks = s.webhook
```

Comment:
If you change the type to a pointer you also have to assign a pointer here. We know that at least one must be set so no need to check if all of them are false to use `nil` instead of `&s.webhook`
```suggestion
	res.Webhooks = &s.webhook
```